guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -1,7 +1,23 @@
 [[Configuring_Provisioning_Resources]]
 == Configuring Provisioning Resources
 
-{ProjectNameX} contains provisioning resources that you can use to create a host.
+{ProjectNameX} contains provisioning resources that you can use to create a host. These include:
+
+. *Domain*. Every host managed by {Project} must be assigned to a domain without exceptions. {Project} can then manage A, AAAA and PTR records when configured appropriately, however if no integration is needed then at least one domain must be created. Domain names are taken into account for naming hosts, e.g. host named "test123" in a domain named "example.com" will get full host name "test123.example.com"
+. *Subnet*. Every host managed by {Project} must be assigned to a subnet without exceptions. {Project} can then manage IPv4 reservations when configured appropriately, however if no integration is needed then at least one subnet must be created. IP address management can be either manual (set to None), DHCP (managed DHCP server is asked for a free IP from its pool) or Database (serial or random IP address book keeping in {Project} database).
+. *Provisioning template*. While it is possible to create static provisioning templates (e.g. kickstart), {Project} comes with many provisioning templates written in ERB (Embedded Ruby) syntax which allows modularization per host or host groups. For example timezone in a kickstart can be different depending on given host parameter. Depending on a provisioning workflow, various combinations needs to be set.
+. *Partition table*. Partition table is a special kind of template used only for rendering disk volume definition part of kickstart, it has the very same syntax (ERB). This allows to define multiple disk layouts and select them per host or host group depending on system purpose.
+. *Installation media*. Installation media represents a parametrized URL to a remote repository with OS packages (e.g. yum or apt).
+ifeval::["{build}" == "foreman"]
+When Katello plugin is installed, packages can be also downloaded from Pulp mirror, in this case installation media is ignored.
+endif::[]
+ifeval::["{build}" == "satellite"]
+. This can be used to install 3rd party software, Red Hat content is delivered through repository syncing feature instead.
+endif::[]
+. *Subscriptions and repositories*. In order to consume Red Hat content, subscription must be uploaded into {ProjectName}, a Product must be present in the subscription and one or more installation Repositories must be enabled and synchronized into the initial lifecycle environment called Library. Optionally, content may be filtered and promoted into other Lifecycle Environments or Content Views.
```

Comment:
Does this apply to Foreman also, @lzap I would have thought this would require an additional plugin. Perhaps we should wrap this in an ifeval also?